conllulex/lexcatter.py

```diff
@@ -45,6 +49,9 @@ def supersenses_for_lexcat(lc):
     "X",
 }
 
+HI_LEXCATS = BASE_LEXCATS
+HI_LEXCATS.add('PART')
```

Comment:
this should be `HI_LEXCATS = BASE_LEXCATS.copy()`, otherwise `'PART'` will also be added to `BASE_LEXCATS`

Reply:
OK, pushed updates to [this commit](https://github.com/nert-nlp/conllulex/pull/4/commits/71f2ba86ff9011974d26e42c3854b2c61b45f0ba)